server/src/main/java/io/crate/expression/scalar/arithmetic/TruncFunction.java

```diff
@@ -54,7 +54,8 @@ public static void register(ScalarFunctionModule module) {
                     NAME,
                     type.getTypeSignature(),
                     returnType.getTypeSignature()
-                ).withForbiddenCoercion(),
+                ).withForbiddenCoercion()
+                    .withFeature(Scalar.Feature.NULLABLE),
```

Comment:
very minor: please fix identation, to match the `.withForbiddenCoersion()`, maybe move the `.withForbiddenCoersion()` to a new line.